Aktas2012/Aktas2012.md

```diff
@@ -0,0 +1,16 @@
+# Aktas 2004
+
+Can B. Aktas. Ph.D. 2012. Impact of lifetime on US residential building LCA results. The International Journal of Life Cycle Assessment https://link.springer.com/article/10.1007%2Fs11367-011-0363-x
```

Comment:
The reference is wrong. It should be more like: *Aktas, C.B. and M.M. Bilec. 2012. Impact of lifetime on US residential building LCA results. The International Journal of Life Cycle Assessment 17(3): 337–349. http://link.springer.com/10.1007/s11367-011-0363-x.*